test/test.go

```diff
@@ -34,6 +34,7 @@ func SetupTestEnvironment() {
 
 func CleanUpDatabase() {
 	database.GetDB().Exec("TRUNCATE TABLE users")
+	database.GetDB().Exec("TRUNCATE TABLE keywords")
```

Comment:
Once you will add the foreign key between users and keywords, this will crash: cannot delete a user that is associated with keywords. (unless you have a config to cascade delete or nullify on dependency deleted)

In either way, I recommend to 
- delete first the keywords
- delete last the users